tests/test_random_number.py

```diff
@@ -0,0 +1,5 @@
+from src.random_number import random_number
+
+def test_random_number() -> None:
+    t = random_number(0, 100)
+    assert t >= 0 and t <= 100
```

Comment:
Volendo potresti mettere questo test in un ciclo giusto per fargli fare un po' di passate e vedere piu' output, ma essendo random in principio non cambia tantissimo